packages/next/src/lib/metadata/generate/opengraph.tsx

```diff
@@ -202,7 +202,11 @@ export function OpenGraphMetadata({
       <Meta property="og:locale" content={openGraph.locale} />
       <Meta property="og:country_name" content={openGraph.countryName} />
       <Meta property="og:ttl" content={openGraph.ttl?.toString()} />
-      <MultiMeta propertyPrefix="og:image" contents={openGraph.images} />
+      <MultiMeta
+        propertyPrefix="og:image"
+        mapKey={(key) => (key === 'og:image:url' ? 'og:image' : key)}
```

Comment:
Can we add a special condition inside `MultiMeta` for this, as it's the only case AFAIK? For example:

```js
property: propertyPrefix === 'og:image' && k === 'url' ? 'og:image' : propertyPrefix + k
```

Instead of a function call for every key.